internal/commands/scan.go

```diff
@@ -384,6 +387,37 @@ func scanShowSubCommand(scansWrapper wrappers.ScansWrapper) *cobra.Command {
 	return showScanCmd
 }
 
+func scanVorpalSubCommand() *cobra.Command {
```

Comment:
FYI, the command will be public 
Sorry if it missed when the requirement came

Reply:
Maybe I misunderstand here, but all of our command functions are private, what makes them visible or not is the fact that we add them to the Cobra command:

scanCmd.AddCommand(
		createScanCmd,
		scanVorpalCmd,
		showScanCmd,
		workflowScanCmd,
		listScansCmd,
		deleteScanCmd,
		cancelScanCmd,
		tagsCmd,
		logsCmd,
		kicsRealtimeCmd,
		scaRealtimeCmd,
	)

(the command itself should be hidden though, right?)

Reply:
REMOVE: hidden: true